iceoryx_binding_c/include/iceoryx_binding_c/publisher.h

```diff
@@ -24,20 +24,29 @@
 /// @brief publisher handle
 typedef struct cpp2c_Publisher* iox_pub_t;
 
+/// @brief options to be set for a publisher
+typedef struct
+{
+    // size of the history chunk queue
+    uint64_t historyCapacity;
+
+    // name of the node the publisher belongs to
+    const char* nodeName;
+} iox_publisher_options_t;
```

Comment:
please create a `iox_pub_init_options` method to initialize all members to default values. This enables us to add additional members to the struct without breaking the code of our users. At least not when used correctly.

Reply:
Can do. We cannot force anyone to use it though.

Reply:
Shroten to iox_pub_options_t